index.d.ts

```diff
@@ -0,0 +1,81 @@
+export type PostPreview = {
```

Comment:
`index.d.ts` is a special kind of file, and you don't need to actually export the types. If you remove the `export`, then you also don't need to explicitly import then to other files, it just works automagically